docs/tutorials/getting-started.md

```diff
@@ -0,0 +1,118 @@
+# Quick Guide
+
+## What you’ll learn
+
+- Deploy the [Content-cache-k8s charm](https://charmhub.io/content-cache-k8s).
+- Relate to [the Hello-kubecon charm](https://charmhub.io/hello-kubecon).
+- Relate to [Ingress](https://kubernetes.io/docs/concepts/services-networking/ingress/#what-is-ingress) by using [NGINX Ingress Integrator](https://charmhub.io/nginx-ingress-integrator/).
+
+Through the process, you'll inspect the Kubernetes resources created, verify the workload state and assign the Content-cache-k8s to serve your Hello-kubecon instance.
+
+## Requirements
+
+- Juju 3 installed.
+- Juju controller and model created.
+- NGINX Ingress Controller. If you're using [MicroK8s](https://microk8s.io/), this can be done by running the command `microk8s enable ingress`. For more details, see [Addon: Ingress](https://microk8s.io/docs/addon-ingress).
+
+For more information about how to install Juju, see [Get started with Juju](https://juju.is/docs/olm/get-started-with-juju).
```

Comment:
Should we put that next to `Juju 3 installed` point ? Like we did for Nginx ingress ?

Reply:
I think leaving it as it is (same as in Indico) looks better. I'd prefer to list all the requirements first and then write the "for more information" section

Reply:
Then I suggest to move the part concerning Microk8s and ingress in a "for more information" paragraph for coherence